doc/book/cookbook/setting-module-specific-layout.md

```diff
@@ -0,0 +1,85 @@
+# Setting module-specific Layout
```

Comment:
This title:

- has a mix of cases
- is missing either a plural, or an identifier.

I'd suggest one of:

- Setting A Module-Specific Layout
- Setting Module-Specific Layouts

Reply:
@weierophinney 
> has a mix of cases

Right, because: 

* this first word and a noun must be start with a capital letter
* all other words are written with small letters

_(The documentation standard guide is work in progress.)_